### PR TITLE
UICM-75: Fixed the filtering issue in select-drug directive

### PR DESCRIPTION
**Pull request by mhawila**

### CODE REVIEW

File: omod/src/main/webapp/resources/scripts/directives/select-drug.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ angular.module('uicommons.widget.select-drug', [ 'drugService', 'ui.bootstrap' ]
                 }
             },
             template: '<input type="text" id="{{ inputId }}" ng-model="ngModel" ng-blur="verify()" ' +
-                'typeahead="drug as drug.display for drug in search($viewValue) | filter:$viewValue" ' +
+                'typeahead="drug as drug.display for drug in search($viewValue) " ' +
                 'typeahead-on-select="onSelect($item, $model, $label)" ' +
                 'typeahead-editable="false" autocomplete="off" placeholder="{{ placeholder }}" autocomplete="off" ' +
                 'ng-required="{{ required }}" size="{{ size }}" ' +
```
Comment (wluyima) [inline omod/src/main/webapp/resources/scripts/directives/select-drug.js]: From [this](https://talk.openmrs.org/t/options-for-openmrs-drug-management/1719/32) talk topic, seems like there is more to be done.

Comment (mhawila) [reply omod/src/main/webapp/resources/scripts/directives/select-drug.js]: There isn't more on this module. The other piece of work is supposed to be done on the order entry ui module.

Comment (mhawila) [reply omod/src/main/webapp/resources/scripts/directives/select-drug.js]: @wluyima even if this code works it is redundant. The results sent from the backend are already filtered based on the typed in characters, why filter the results again?

Comment (wluyima) [reply omod/src/main/webapp/resources/scripts/directives/select-drug.js]: Makes sense

**Pull request merged**